### PR TITLE
Feature dashboard

### PR DESCRIPTION
**Pull request by gpicart**

### CODE REVIEW

File: cms/src/common/business-form/business-form.jsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+import {Business} from "../../models/business";
+
+const RenderField = ({field, readOnly, value, onChangeFn, required = false}) => (
+  <>
+    {readOnly ?
+      <p className={style.readValue}>{value}</p>
+      :
+      <input id={`business-${field}`} onChange={onChangeFn} required={required}
+             value={value} name={field} className={style.input}/>
+    }
```
Comment (mlvieras) [inline cms/src/common/business-form/business-form.jsx]: Don't want you to change this, but inputs have a `readOnly` prop that makes them read only. Maybe you wanted it not to be an input, which is fine :+1:

File: cms/src/common/business-form/business-form.jsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
+import style from "./business-form.module.scss";
+import React from "react";
+import PropTypes from "prop-types";
+import {Business} from "../../models/business";
+
+const RenderField = ({field, readOnly, value, onChangeFn, required = false}) => (
```
Comment (mlvieras) [inline cms/src/common/business-form/business-form.jsx]:
```suggestion
const Field = ({field, readOnly, value, onChangeFn, required = false}) => (
```

File: cms/src/common/business-form/business-form.jsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    {readOnly ?
+      <p className={style.readValue}>{value}</p>
+      :
+      <input id={`business-${field}`} onChange={onChangeFn} required={required}
+             value={value} name={field} className={style.input}/>
+    }
+  </>
+);
+
+const RenderFieldCheckbox = ({field, readOnly, value, onChangeFn, stateValue, label}) => (
```
Comment (mlvieras) [inline cms/src/common/business-form/business-form.jsx]:
```suggestion
const FieldCheckbox = ({field, readOnly, value, onChangeFn, stateValue, label}) => (
```

File: cms/src/models/business.js [hunk 4]
```diff
@@ -14,8 +14,12 @@ class Business {
     this.deliveryZone = params.delivery_zone;
     this.tags = params.tags;
     this.time = params.available_hours;
-
   }
 }
 
+Business.typeConstants = {
```
Comment (mlvieras) [inline cms/src/models/business.js]: :+1:

File: cms/src/common/business-form/business-form.jsx (hunk 1 above)
Comment (gpicart) [reply cms/src/common/business-form/business-form.jsx]: Yes, the idea behind of it being a <p> is that it's more flexible if we want to add a better style to it.

**Review by mlvieras — APPROVED** [review]

**Pull request merged**